components/Extensions/samples/ShadowAnimations.md

```diff
@@ -12,7 +12,7 @@ issue-id: 0
 icon: Assets/ShadowAnimation.png
 ---
 
-Either type of Attached Shadow can be easily animated using the Toolkit's [`AnimationSet`](../animations/AnimationSet.md) api. These provide an easy XAML based way to animate a wide variety of elements, including a variety of shadow properties. They can also be animated with any other composition animation technique in code-behind as well using either the [`AnimationBuilder`](../animations/AnimationBuilder.md) or built-in composition animations.
+Either type of Attached Shadow can be easily animated using the Toolkit's [`AnimationSet`](../animations/AnimationSet.md) api. These provide an easy XAML based way to animate a wide variety of elements, including a variety of shadow properties. They can also be animated with any other composition animation technique in code-behind as well using either the [`AnimationBuilder`](../Behaviors/AnimationSet.md) or built-in composition animations.
```

Comment:
Maybe in the future we should think about an analyzer for our markdown stuff we load anyway that looks for relative links and sees if the files exist in the repo? Let's just create an issue from this comment in the tooling repo for tracking later.

Reply:
See https://github.com/CommunityToolkit/Tooling-Windows-Submodule/issues/106